### PR TITLE
The GUI is opposite of the API - This is easier

### PR DESCRIPTION
**Pull request by gungazoo**
This PR closes #

The changes in this PR are as follows:

*  Allows more selections for DRS
*  Matches the numbers shown in the vCenter GUI

I have read the [contributor guidelines](CONTRIBUTING.md) and have completed the following:

* [x ] **Formatted the code** using VSCode default formatter for PowerShell.
* [x ] **Tested the code** end-to-end against an SDDC.
* [ x] **Documented the functions** using standard PowerShell markup and applied `AVSAttribute` to newly exported functions.

### CODE REVIEW

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 1]
```diff
@@ -2661,11 +2661,19 @@ function Set-CustomDRS {
         [Parameter(Mandatory = $true)]
         [String]$ClustersToChange,
         [Parameter(Mandatory = $true,
-            HelpMessage = "The DRS setting. Default of 3 or more conservative of 4.")]
+            HelpMessage = "The DRS setting. Default of 3 or more conservative of 2 or less conservative 4.")]
         [ValidateRange(3, 4)]
-        [int] $Drs
+        [int] $DrsChoice
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:2666]: Renaming a required parameter is a breaking change, need to either bump the major or keep the name.

Comment (gungazoo) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1]: Sorry -- Didn't know. I've changed them required param back.

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 2]
```diff
@@ -2661,15 +2661,23 @@ function Set-CustomDRS {
         [Parameter(Mandatory = $true)]
         [String]$ClustersToChange,
         [Parameter(Mandatory = $true,
-            HelpMessage = "The DRS setting. Default of 3 or more conservative of 4.")]
+            HelpMessage = "The DRS setting. Default of 3 or more conservative of 2 or less conservative 4.")]
         [ValidateRange(3, 4)]
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:2665]: Does this range validation make sense if 2 and 1 are handled in the switch?

Comment (gungazoo) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:2665]: Before they could only go one step more conservative. Just letting them know that they have more options.

Comment (et1975) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:2665]: I mean in line 2665

Comment (gungazoo) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1]: Of course you are correct! I completely overlooked that. I've pushed the change.

**Review by et1975 — APPROVED** [review]

**Pull request merged**